spglm/glm.py

```diff
@@ -245,28 +246,6 @@ class GLMResults(LikelihoodModelResults):
         normalized_cov_params   : array
                                 k*k, approximates [X.T*X]-1
 
-    Examples
```

Comment:
This example was deleted on purpose?

Reply:
Yes. Line 265 will run into an error. My understanding is that result1 (GLMResults instance) does not have attribute `predy`.

Reply:
I could be wrong. Feel free to add it back.